### DISCUSSION TITLE
Presentación de los vídeos de asciiinema

### DISCUSSION THREAD

**Discussion by carlosal1015** [cpp-review-dune/introductory-review#9] [Show and tell]
Por favor, coloquen sus enlaces de las grabaciones de prueba en `asciinema`.

```console
$ asciinema rec -c bash
```

**Comment by carlosal1015** [comment]
- [Cmake sample part 1](https://asciinema.org/a/468534)
- [Cmake sample part 2](https://asciinema.org/a/468537)

**Reply by carlosal1015** [reply]
Vamos @dgcamacho 😄

**Comment by carlosal1015** [comment]
![Screenshot_select-area_20220213071600](https://user-images.githubusercontent.com/21283014/153752584-e3a11d30-eb39-40c0-ab74-840ef3ee9657.png)

**Reply by andromeda2075** [reply]
Done it! 
https://asciinema.org/a/470253 ( enlace que también está en discusiones)
